src/main/java/com/sololegends/runelite/VarlamoreHouseThievingPlugin.java

```diff
@@ -199,6 +199,13 @@ public void onGameTick(GameTick event) {
 		} else {
 			done_stealing_notified = false;
 		}
+
+		// Time since last distraction notification check
+		long timeSinceLastDistraction = NextUpOverlayPanel.sinceDistraction();
+
+		if (config.notifyOnTimeSinceDistraction() == timeSinceLastDistraction && config.notifyOnTimeSinceDistraction() > 0) {
```

Comment:
Flow the second conditional, including operator, to the following line indented by on level. 
Otherwise the line length is too long.